<?xml version="1.0"?>
<launch>
<group ns="drone17">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
  <arg name="name" default="drone17"/>
  <arg name="tf_prefix" value="drone17"/>
  <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor_downward_cam_green.gazebo.xacro"/>
  <arg name="x" default="0.0"/>
  <arg name="y" default="12.0"/> 
  <arg name="z" default="0.1"/>
  <!-- <arg name="x" default="6.0"/>
  <arg name="y" default="15.0"/> 
  <arg name="z" default="0.2"/> -->
  </include>
  </group>
</launch>